lib/python/Components/VfdSymbols.py

```diff
@@ -1,326 +1,263 @@
+						fileWriteLine(self.recPath, value, source=MODULE_NAME)
+				case 5:
+					self.blink = not self.blink
+					value = "0xffffffff"
+					if recordings:
+						if self.blink:
+							value = "0x00000000"
+							self.led = True
+						else:
+							self.led = False
+					fileWriteLine(self.recPath, value, source=MODULE_NAME)
+				case 6:
+					self.blink = not self.blink
+					if recordings:
+						if self.blink:
+							value = "0"
+							self.led = True
+						else:
+							value = config.usage.lcd_ledstandbycolor.value if Screens.Standby.inStandby else config.usage.lcd_ledpowercolor.value
+							self.led = False
+					else:
+						value = config.usage.lcd_ledstandbycolor.value if Screens.Standby.inStandby else config.usage.lcd_ledpowercolor.value
+					fileWriteLine(self.recPath, value, source=MODULE_NAME)
+				case 7:
+					if recordings:
+						fileWriteLine("/proc/stb/lcd/symbol_recording", "1", source=MODULE_NAME)
+						fileWriteLine("/proc/stb/lcd/symbol_record_1", "1", source=MODULE_NAME)
+						if recordings == 1:
+							fileWriteLine("/proc/stb/lcd/symbol_record_2", "0", source=MODULE_NAME)
+						elif recordings >= 2:
+							fileWriteLine("/proc/stb/lcd/symbol_record_2", "1", source=MODULE_NAME)
+					else:
+						fileWriteLine("/proc/stb/lcd/symbol_recording", "0", source=MODULE_NAME)
+						fileWriteLine("/proc/stb/lcd/symbol_record_1", "0", source=MODULE_NAME)
+						fileWriteLine("/proc/stb/lcd/symbol_record_2", "0", source=MODULE_NAME)
+
+	def setResolution(self):
+		path = self.ledConfig.get("symbol_hd")
+		if path:
+			info = self.service and self.service.info()
+			if info:
+				fileWriteLine(path, "1" if int(info.getInfo(iServiceInformation.sVideoWidth)) >= 1280 else "0", source=MODULE_NAME)
+
+	def setSubtitle(self):
+		subtitlePath = self.ledConfig.get("symbol_subtitle")
+		smartcardPath = self.ledConfig.get("symbol_smartcard")
+		if subtitlePath or smartcardPath:
+			subtitle = self.service and self.service.subtitle()
+			subtitleList = subtitle and subtitle.getSubtitleList()
+			if subtitleList:
+				subtitles = len(subtitleList)
+				if subtitlePath:
+					fileWriteLine(subtitlePath, "1" if subtitles else "0", source=MODULE_NAME)
+				else:
+					fileWriteLine(smartcardPath, "1" if subtitles else "0", source=MODULE_NAME)
+			else:
+				if subtitlePath:
+					fileWriteLine(subtitlePath, "0", source=MODULE_NAME)
+				else:
+					fileWriteLine(smartcardPath, "0", source=MODULE_NAME)
+
+	def setTeletext(self):
+		path = self.ledConfig.get("symbol_teletext")
+		if path:
+			info = self.service and self.service.info()
+			if info:
+				fileWriteLine(path, "1" if int(info.getInfo(iServiceInformation.sTXTPID)) != -1 else "0", source=MODULE_NAME)
+
+	def setTimer(self):
+		path = self.ledConfig.get("symbol_timer")
+		if path:
+			fileWriteLine(path, "1" if self.session.nav.RecordTimer.getNextRecordingTime() > 0 else "0", source=MODULE_NAME)
+
+
+def SymbolsCheck(session, **kwargs):
+	global vfdsymbolsupdater
```

Comment:
Where is this variable used?

If possible please use camelCase.  That is, this should be "vfdSymbolsUpdater".